covid_files.R

```diff
@@ -0,0 +1,205 @@
+locus<-paste('/Volumes/IDGenomics_NAS/COVID/daily_metadata/lims/',NGS_covid,sep=""); locus
+NGS_covid <- read.table(locus,header=TRUE,sep=",", stringsAsFactors=FALSE,comment.char="",nrows=500000, fill = T)
+dim(NGS_covid)
+NGS_covid<-NGS_covid[,c("sampleNumber","lastName","firstName","DOB","collectionDate")]
+NGS_covid$sampleNumber<-as.character(NGS_covid$sampleNumber); 
+
+# (2) from All_nocovid_NoNGS & and NGS_covid
+cat(crayon::bold("2. From All_nocovid_NoNGS","\n"))
+All_nocovid<-rownames(fileSnapshot("/Volumes/IDGenomics_NAS/COVID/daily_metadata/lims/",
+                                   pattern=glob2rx("All*.csv"))$info[which.max(fileSnapshot("/Volumes/IDGenomics_NAS/COVID/daily_metadata/lims/",
+                                                                                            pattern=glob2rx("All*.csv"))$info$mtime),])
+locus<-paste('/Volumes/IDGenomics_NAS/COVID/daily_metadata/lims/',All_nocovid,sep="")
+All_ncovid_NoNGS <- read.table(locus,header=TRUE,sep=",", stringsAsFactors=FALSE,comment.char="", fill = T,quote="\"")
+All_ncovid_NoNGS<-All_ncovid_NoNGS[,c("sampleNumber","lastName","firstName","DOB","collectionDate")]
+
+# Generating a LIMS-DB for this run
+# ========================
+LIMS<-rbind(NGS_covid,All_ncovid_NoNGS)
+dim(LIMS)
+dim(M)
+ngs<-merge(M,LIMS, by.x ="Sample_Accession",by.y="sampleNumber", x.all = T) # aqui pasa algo loquisimo
+dim(ngs)
+ngs<-cbind(ngs,'Name of facility performing the test'='UPHL', 'ordering facility' ='UHPL',
+           'Name or description of the test ordered'="SARS-CoV-2 Whole Genome Sequencing",
+           'lab test date'=date,'lab test status' ="preliminary")
+ngs_file<-select(ngs,"lastName",	"firstName",	"DOB",	"Name of facility performing the test",
+                 "Name or description of the test ordered","collectionDate", "Sample_Accession",
+                 "lab test date","lab test status","lineage","ordering facility")
+ngs_file[is.na(ngs_file)] <- ""
+colnames(ngs_file)[7]<-c("sampleNumber");colnames(ngs_file)[10]<-c("Test result")
+head(ngs_file)
+dim(ngs_file)
+
+# Storing ngs file at /Volumes/NGS/Analysis/covidseq/$runName
+# ==========================
+d=paste(runPath,'Rsumcovidseq',sep="/")
+dir.create(file.path(d,"" ), recursive = T)
+LL<-paste(substring(Sys.time(), 1,10),substring(Sys.time(), 12,13),
+          substring(Sys.time(),15,16),substring(Sys.time(), 18,19),sep = "-")
+dx<- paste(paste('ngs_file',args[1],sep="_"),LL,".csv",sep="")
+dx<-paste(d,dx, sep ='/');write.csv(ngs_file,dx,row.names=F)
+cat(crayon::bold("ngs file is complete and can be found at",d,"\n"))
+
+# Summary file
+# ======================
+clade = select(read.table(file = paste(runPath,"CecretPangolin/nextclade/nextclade.tsv",
+                                       sep ="/"), sep = '\t', header = T),seqName,clade); 
+clade<-clade %>% mutate_at("seqName", str_replace, var, "")
```

Comment:
It doesn't look like the nextclade clade is used anywhere. Am I missing something?